lms/static/js/lang-edx.js

```diff
@@ -0,0 +1,47 @@
+var edx = edx || {},
+    Language = (function($) {
+        'use strict';
+        var lang_edx = {
+
+            init: function() {
+                lang_edx.listenForChange();
+            },
+
+            /**
+             * Listener on changing language from selector.
+             * Send an ajax request to save user language preferences.
+             */
+            listenForChange: function() {
```

Comment:
Can we be more specific about the type of change we are listening for here?  Perhaps listenForLanguagePreferenceChange or something along those lines.